pyproject.toml

```
@@ -0,0 +1,24 @@
+[project]
+name = "rapyuta-io-sdk-v2"
+version = "0.1.0"
```

todo: Refer the yaml in the CLI. We'll have to set it up such that the version is read from `__init__.py` from an attribute named `__version__`. 

We will use the semantic release process that will automatically update the version.
```suggestion
dynamic = [version]
```